

def error_msg(message: str) -> str:
    raise Exception(message)